### DISCUSSION TITLE
Any typescript examples?

### DISCUSSION THREAD

**Discussion by cmacdonnacha** [TanStack/query#1221] [Q&A]
Hey everyone,

I can't seem to find any Typescript examples? Just wondering if there are any?

**Comment by TkDodo** [comment]
I don’t think so, but I thought about adding one for v3. Anything particular you would like to see there?

**Reply by cmacdonnacha** [reply]
The [basic](https://react-query.tanstack.com/docs/examples/basic) and [optimistic updates](https://react-query.tanstack.com/docs/examples/optimistic-updates) examples would be nice 👍
